azext_edge/edge/_help.py

```diff
@@ -85,7 +85,7 @@ def load_iotops_help():
 
         - name: Constrain data capture on a specific service as well as producing the bundle in a custom output dir.
           text: >
-            az iot ops support create-bundle --ops-service opcua --bundle-dir ~/ops
+            az iot ops support create-bundle --ops-service connectors --bundle-dir ~/ops
```

Comment:
Is this already all connectors (1p, 3p, media, onvif, etc)? or just opcua connectors?

Reply:
furthermore - is it ok to put everything into the same folder? I can see people wanting to organize it by connector type (especially if it becomes hectic)

Reply:
talked offline, for now let's keep it as it is for this iteration